site/kubernetes/operator/using-operator.md

```diff
@@ -382,6 +382,10 @@ spec:
       memory: 2Gi
 </pre>
 
+**Note:** It's possible for RabbitMQ to temporary exceed its memory high-water mark which could cause immediate OOM kill.
+cluster operator sets a 20% memory headroom (with a max value of 2GB) to prevent this from happening.
```

Comment:
To prevent this from happening, the cluster operator sets memory headroom of 20% (with a max value of 2GB).